packages/shared/src/modules/categories/libs/types/categories-get-by-id-request-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+type CategoriesGetByIdRequestDto = {
+	categoryIds: string;
```

Comment:
I can see that there is a single string there. Shouldn't it be single `categoryId` without `s` then? Also, all ids we have in the app are numbers.

Reply:
It's beause it will be  passed as a query parameter via URL. Ususaly I would pass data through request's body, but In previous Code reviews iIthink I was told that GET requests shouldn't have a body.  `categoryIds` are parsed into an array of numbers at service level

Reply:
I think it would be better to call this type `CategoriesGetByIdsRequestDto`. And in controller, service and repository rename method from `getById` or `findById` to `getByIds` or `findByIds`. Because when you write get something by id, it seems that it should return one entity by one id.